test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -2142,8 +2132,6 @@ def create_temp_dir(self, hostname=None, suffix='', prefix='PtlPbs',
                           recursive=True, uid=uid, gid=gid, level=level)
             if mode is not None:
                 self.chmod(hostname, path=tmpdir2, mode=mode, runas=asuser)
-            else:
```

Comment:
mode can't really be None now, unless of course someone passes in None.  Also, why do all the chmoding to 0o755 instead of mode?  It defaults to 0o755, so you should only really have to do the chmod once.

Reply:
Right, someone can pass mode as None that is why I left it as is. 
The reason I'm explicitly using chmod 755 instead of using mode is that if someone gives a very restrictive mode then the function might not even be able to do a run_copy. This is the reason I create a directory explicitly as 755 and then do a copy and then chmod it based on what was given.

Reply:
I wouldn't bother checking for None.  The type of this is an octal integer,  If someone passes None, it shouldn't work and crapping out on chmod() is fine.